src/drive/mobile/modules/settings/components/DebugTools.jsx

```diff
@@ -10,39 +10,45 @@ const Checkbox = ({ onChange, value, title }) => (
   </div>
 )
 
-class DebugTools extends Component {
-  sendSentryException() {
-    logException('a debug exception')
-  }
+const sendSentryException = () => logException('a debug exception')
 
-  sendSentryMessage() {
-    logInfo('a debug message')
-  }
+const sendSentryMessage = () => logInfo('a debug message')
 
-  render() {
-    return (
-      <div>
-        <h4>Sentry</h4>
-        <Button
-          onClick={() => this.sendSentryException()}
-          label="send exception"
-        />
-        <Button onClick={() => this.sendSentryMessage()} label="send message" />
-        <h4>Offline</h4>
-        <Checkbox
-          title="First Replication"
-          value={this.props.firstReplication}
-          onChange={this.props.setFirstReplication}
-        />
-        <Checkbox
-          title="Offline"
-          value={this.props.offline}
-          onChange={this.props.setOffline}
-        />
-        <hr />
-      </div>
-    )
-  }
+const getPersistedDoctypes = () => {
+  return JSON.parse(
+    window.localStorage.getItem('cozy-client-pouch-link-synced')
+  )
+}
+const getPersistedWarmupedQueries = () => {
+  return window.localStorage.getItem('cozy-client-pouch-link-warmupedqueries')
+}
+const DebugTools = ({ offline, setOffline }) => {
+  return (
+    <div>
+      <h4>Sentry</h4>
+      <Button onClick={() => sendSentryException()} label="send exception" />
+      <Button onClick={() => sendSentryMessage()} label="send message" />
+      <h4>Offline</h4>
+      <Checkbox title="Offline" value={offline} onChange={setOffline} />
+      <h4
+        onClick={() => {
+          const doctypes = getPersistedDoctypes()
+          alert(doctypes)
+        }}
+      >
+        Doctypes Persistés (aka synchronisés au moins une fois)
+      </h4>
+      <h4
+        onClick={() => {
+          const queries = getPersistedWarmupedQueries()
+          alert(queries)
```

Comment:
In drive, you can't copy anything, we have `user-select: none` (https://github.com/cozy/cozy-drive/blob/master/src/drive/styles/mobile.styl#L4). We should have a good reason to have it years ago.

The debug is only there to display null or something else, since we can't have other text displayed. 

I want to enhance this debug panel, but let's do that in a next PR in a few days